bazel/deps.bzl

```diff
@@ -26,6 +27,7 @@ BF_SDE_PI_VER = {
     "9_3_2": "4546038f5770e84dc0d2bba90f1ee7811c9955df",
     "9_4_0": "4546038f5770e84dc0d2bba90f1ee7811c9955df",
     "9_5_0": "4546038f5770e84dc0d2bba90f1ee7811c9955df",
+    "9_11_0": "4546038f5770e84dc0d2bba90f1ee7811c9955df",
```

Comment:
Seems like its just repeating the SHA. Candidate for cleanup in future.

Reply:
It gets deleted entirely further down the line.